libmscore/tempotext.cpp

```diff
@@ -338,33 +338,51 @@ void TempoText::layout()
       adjustReadPos();
       }
 
+QString TempoText::duration2userName(const TDuration t)
+      {
+      QString dots;
+      switch (t.dots()) {
+            case 1: dots = tr("Dotted %1").arg(t.durationTypeUserName());
+                  break;
+            case 2: dots = tr("Double dotted %1").arg(t.durationTypeUserName());
+                  break;
+            case 3: dots = tr("Triple dotted %1").arg(t.durationTypeUserName());
+                  break;
+            default:
+                  dots = t.durationTypeUserName();
+                  break;
+            }
+      return dots;
+      }
+
 //---------------------------------------------------------
 //   accessibleInfo
 //---------------------------------------------------------
 
 QString TempoText::accessibleInfo() const
       {
-      TDuration t;
-      int len;
-      int x = findTempoDuration(plainText(), len, t);
-      if (x != -1) {
-            QString dots;
-
-            switch (t.dots()) {
-                  case 1: dots = tr("Dotted %1").arg(t.durationTypeUserName());
-                        break;
-                  case 2: dots = tr("Double dotted %1").arg(t.durationTypeUserName());
-                        break;
-                  case 3: dots = tr("Triple dotted %1").arg(t.durationTypeUserName());
-                        break;
-                  default:
-                        dots = t.durationTypeUserName();
-                        break;
+      TDuration t1;
+      TDuration t2;
+      int len1;
+      int len2;
+      QString text = plainText();
+      QString firstPart = text.split(" = ").first();
+      QString secondPart = text.split(" = ").back();
+      int x1 = findTempoDuration(firstPart, len1, t1);
+      int x2 = -1;
+      if (_relative)
+            x2 = findTempoDuration(secondPart, len2, t2);
+
+      if (x1 != -1) {
+            QString dots1;
+            QString dots2;
+            dots1 = duration2userName(t1);
+            if (x2 != -1) {
+                  dots2 = duration2userName(t2);
+                  return QString("%1: %2 %3 = %4 %5").arg(Element::accessibleInfo()).arg(dots1).arg(tr("note")).arg(dots2).arg(tr("note"));
                   }
-
-            QString bpm = plainText().split(" = ").back();
-
-            return QString("%1: %2 %3").arg(Element::accessibleInfo()).arg(dots).arg(tr("note = %1").arg(bpm));
+            else
+                  return QString("%1: %2 %3").arg(Element::accessibleInfo()).arg(dots1).arg(tr("note = %1").arg(secondPart));
```

Comment:
I think also it is better, but had it in there for compatibility but since it is 3.0 we should just do it!